portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,46 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width:100%;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+.display-4{
```

Comment:
Maybe give this a more specific name so looking just at the CSS someone would be able to have a basic understanding of what it is effecting in the html.

Reply:
This is a class from bootstrap that I am trying to override, but I can add something else